gravitee-apim-console-webui/src/management/api/creation-v4/steps/step-2-entrypoints/step-2-entrypoints-2-config.component.html

```diff
@@ -16,85 +16,87 @@
 
 -->
 <mat-card class="api-creation-v4__step">
-  <div class="api-creation-v4__step__header">
-    <div class="api-creation-v4__step__header__step-number">Step 2</div>
-    <div class="api-creation-v4__step__header__subtitle">Configure your API entrypoints</div>
-    <p class="api-creation-v4__step__header__paragraph-light">Configure how your users will consume your API</p>
-  </div>
+  <mat-card-content>
+    <div class="api-creation-v4__step__header">
+      <div class="api-creation-v4__step__header__step-number">Step 2</div>
+      <div class="api-creation-v4__step__header__subtitle">Configure your API entrypoints</div>
+      <p class="api-creation-v4__step__header__paragraph-light">Configure how your users will consume your API</p>
+    </div>
 
-  <form *ngIf="formGroup" [formGroup]="formGroup" (ngSubmit)="save()">
-    <div id="http-listeners" class="api-creation-v4__form-container" *ngIf="hasHttpListeners">
-      <h3>Configure common entrypoints fields</h3>
+    <form *ngIf="formGroup" [formGroup]="formGroup" (ngSubmit)="save()">
+      <div id="http-listeners" class="api-creation-v4__form-container" *ngIf="hasHttpListeners">
+        <h3>Configure common entrypoints fields</h3>
 
-      <div class="step-2-entrypoints-2-config__listeners-context__title">
-        <span class="mat-body-strong">Entrypoints context</span>
+        <div class="step-2-entrypoints-2-config__listeners-context__title">
+          <span class="mat-body-strong">Entrypoints context</span>
 
-        <button
-          id="switchListenerType"
-          mat-button
-          type="button"
-          (click)="this.enableVirtualHost = !this.enableVirtualHost"
-          [disabled]="this.enableVirtualHost && this.domainRestrictions.length > 0"
-        >
-          <ng-container *ngIf="!this.enableVirtualHost"> <mat-icon svgIcon="gio:check"></mat-icon> Enable virtual hosts </ng-container>
-          <ng-container *ngIf="this.enableVirtualHost"> <mat-icon svgIcon="gio:cancel"></mat-icon> Disable virtual hosts </ng-container>
-        </button>
+          <button
+            id="switchListenerType"
+            mat-button
+            type="button"
+            (click)="this.enableVirtualHost = !this.enableVirtualHost"
+            [disabled]="this.enableVirtualHost && this.domainRestrictions.length > 0"
+          >
+            <ng-container *ngIf="!this.enableVirtualHost"> <mat-icon svgIcon="gio:check"></mat-icon> Enable virtual hosts </ng-container>
+            <ng-container *ngIf="this.enableVirtualHost"> <mat-icon svgIcon="gio:cancel"></mat-icon> Disable virtual hosts </ng-container>
```

Comment:
The ng-container break the design I think. We should change for something like this
```
    <button
            id="switchListenerType"
            mat-button
            type="button"
            (click)="this.enableVirtualHost = !this.enableVirtualHost"
            [disabled]="this.enableVirtualHost && this.domainRestrictions.length > 0"
          >
            <mat-icon [svgIcon]="enableVirtualHost ? 'gio:cancel' : 'gio:check'"></mat-icon>
            {{ enableVirtualHost ? 'Disable virtual hosts' : 'Enable virtual hosts' }}
          </button>
```

<img width="500" alt="Capture d’écran 2024-02-15 à 16 29 26" src="https://github.com/gravitee-io/gravitee-api-management/assets/25704259/543da4e9-9851-4c23-8eac-02a61460e559">

Reply:
👌